internal/dumpfile/legacy_parse_test.go

```diff
@@ -0,0 +1,284 @@
+package dumpfile
+
+import (
+	"path/filepath"
+	"reflect"
+	"testing"
+)
+
+type testvector struct {
+	filename   string
+	tc         *Content
+	parseerror string
+}
+
+func TestLegacyLoad(t *testing.T) {
+
+	tests := []testvector{
+		{
+			filename:   filepath.Join("legacydumpfiles", "nothere"),
```

Comment:
The [go tool](https://golang.org/cmd/go/) ignores directories named "testdata". Either `legacydumpfiles` should be moved into `testdata` directory or rename it to `testdata`. I'd use `testdata/legacy` (since it's already clear we're dealing with dumpfiles).

Reply:
TIL. Cool. Renamed in subsequent CL.